### ISSUE TITLE
Can we get a say HuggingFace online demo?

### ISSUE THREAD

**Issue by bladedsupernova** [7eu7d7/DreamArtist-sd-webui-extension#19]
Please?

**Comment by AK391** [comment]
@7eu7d7 we can help with hosting a demo on Hugging Face

**Comment by bladedsupernova** [comment]
What can I do to help? (Do you need me? I'm not experienced with HuggingFace or code but can do very simple things)

**Comment by bladedsupernova** [comment]
(or alternatively, how to obtain a colab notebook ?)

**Comment by AK391** [comment]
@bladedsupernova yes i think a colab notebook with gradio demo would be a good idea for training, for example like this but with the gradio demo for training https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/sd_dreambooth_training.ipynb

**Comment by bladedsupernova** [comment]
Oh I mean the pre-trained one, so I can try the anime img2img kaggle notebook

**Comment by bladedsupernova** [comment]
ok i believe my friend got it working for me, it was as simple as going to a github page and using the !git link and replacign that in the link below

replace the !git clone line with this:
!git clone https://github.com/AUTOMATIC1111/stable-diffusion-webui

from here:
https://www.kaggle.com/code/miolovers1/stable-diffusion-automatic1111

**Comment by AK391** [comment]
@7eu7d7 you can host the web ui for training on huggingface see https://huggingface.co/docs/hub/spaces-sdks-gradio then users can duplicate your space to start training
